projects/Generic/linux/linux.x86_64.conf

```diff
@@ -1018,9 +1018,9 @@ CONFIG_NETFILTER_FAMILY_BRIDGE=y
 # CONFIG_NETFILTER_NETLINK_QUEUE is not set
 CONFIG_NETFILTER_NETLINK_LOG=m
 # CONFIG_NETFILTER_NETLINK_OSF is not set
-CONFIG_NF_CONNTRACK=m
```

Comment:
This one was originally a Module, but later complained it needed to be compiled into the kernel, probably caused by some other compiled netfilter module.